cosmoshub/assetlist.json

```diff
@@ -730,6 +730,64 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.svg"
       }
+    },
+    {
+      "description": "OPHIR is a Cosmos Treasury DAO",
+      "extended_description": "The ancient city of gold",
+      "denom_units": [
+        {
+          "denom": "ibc/B26F762ED6D20D0D5305FE1870A476EBCB95127C10199F3CB16E69D893E9F775",
+          "exponent": 0,
+          "aliases": [
+            "transfer/channel-141/transfer/channel-642/factory/migaloo1t862qdu9mj5hr3j727247acypym3ej47axu22rrapm4tqlcpuseqltxwq5/ophir"
+          ]
+        },
+        {
+          "denom": "OPHIR",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/B26F762ED6D20D0D5305FE1870A476EBCB95127C10199F3CB16E69D893E9F775",
+      "name": "Ophir DAO",
+      "display": "OPHIR",
+      "symbol": "OPHIR",
+      "traces": [
+        {
```

Comment:
the order of traces is backwards. You can either put the most recent hop last, or only include the last one.